modules/@apostrophecms/admin-bar/ui/apos/components/TheAposContextTitle.vue

```diff
@@ -31,7 +31,6 @@
           :button="draftButton"
           :menu="draftMenu"
           :disabled="hasCustomUi || isUnpublished"
-          menu-offset="13, 10"
```

Comment:
I removed all of them because they were messing up the positions (it's nicely computed and should not need this).

Reply:
This UI is misplaced. We are setting `menu-placement="bottom-end"` which should align the popup underneath the element ('bottom', working) and at the end of the x-axis (not working)
![image](https://github.com/apostrophecms/apostrophe/assets/1889830/2e81fa29-d53c-4248-94d5-e778cbdcabcf)

Reply:
It's actually the center of the button that triggers it some I'm not shocked.
![image](https://github.com/apostrophecms/apostrophe/assets/17548370/8f3d129c-0d60-48ab-8252-09cbc0fb97e0)

Reply:
Also `bottom-end` means that the arrow is positioned `bottom-end` which is the case here.